charts/trino/templates/deployment-worker.yaml

```
@@ -22,6 +22,7 @@ spec:
     metadata:
       annotations:
         checksum/worker-config: {{ include (print $.Template.BasePath "/configmap-worker.yaml") . | sha256sum }}
+        checksum/catalog-config: {{ include (print $.Template.BasePath "/configmap-catalog.yaml") . | sha256sum }}
```

Nit: let's keep them in the same order as in `deployment-coordinator.yaml`
```suggestion
        checksum/catalog-config: {{ include (print $.Template.BasePath "/configmap-catalog.yaml") . | sha256sum }}
        checksum/worker-config: {{ include (print $.Template.BasePath "/configmap-worker.yaml") . | sha256sum }}
```